momento-sdk/src/main/java/momento/sdk/CacheClient.java

```diff
@@ -1115,393 +1115,453 @@ public CompletableFuture<CacheSortedSetRemoveElementsResponse> sortedSetRemoveEl
+   * @return Future containing the result of the list push front operation: {@link
+   *     CacheListPushFrontResponse.Success} or {@link CacheListPushFrontResponse.Error}.
    */
   public CompletableFuture<CacheListPushFrontResponse> listPushFront(
-      String cacheName,
-      String listName,
-      byte[] value,
-      int truncateBackToSize,
+      @Nonnull String cacheName,
+      @Nonnull String listName,
+      @Nonnull byte[] value,
+      @Nullable Integer truncateBackToSize,
       @Nullable CollectionTtl ttl) {
     return scsDataClient.listPushFront(cacheName, listName, value, truncateBackToSize, ttl);
   }
 
   /**
-   * Pushes a value to the front of the list.
+   * Pushes an value to the front of a list. Refreshes the list with the client's default TTL.
    *
-   * @param cacheName Name of the cache to store the value in
-   * @param listName The list in which the value is to be added.
-   * @param value The element to add to the list.
+   * @param cacheName The cache containing the list.
+   * @param listName The list in which to add the value.
+   * @param value The value to add to the list.
    * @param truncateBackToSize If the list exceeds this length, remove excess from the front of the
-   *     list. Must be positive.
-   * @return Future containing the result of the list push front operation.
+   *     list. Must be positive. Will not truncate if not provided.
+   * @return Future containing the result of the list push front operation: {@link
+   *     CacheListPushFrontResponse.Success} or {@link CacheListPushFrontResponse.Error}.
    */
   public CompletableFuture<CacheListPushFrontResponse> listPushFront(
-      String cacheName, String listName, byte[] value, int truncateBackToSize) {
+      @Nonnull String cacheName,
+      @Nonnull String listName,
+      @Nonnull byte[] value,
+      @Nullable Integer truncateBackToSize) {
     return scsDataClient.listPushFront(cacheName, listName, value, truncateBackToSize, null);
   }
 
   /**
-   * Removes value from the given list.
+   * Removes an value from a list.
    *
    * @param cacheName The cache containing the list.
-   * @param listName The list in which the value is to be removed.
-   * @param value The element to add to the list.
-   * @return Future containing the result of the list remove value operation.
+   * @param listName The list to remove the value from.
+   * @param value The value to remove.
+   * @return Future containing the result of the list remove value operation: {@link
+   *     CacheListRemoveValueResponse.Success} or {@link CacheListRemoveValueResponse.Error}.
    */
   public CompletableFuture<CacheListRemoveValueResponse> listRemoveValue(
-      String cacheName, String listName, String value) {
+      @Nonnull String cacheName, @Nonnull String listName, @Nonnull String value) {
     return scsDataClient.listRemoveValue(cacheName, listName, value);
   }
 
   /**
-   * Removes value from the given list.
+   * Removes an value from a list.
```

Comment:
```suggestion
   * Removes a value from a list.
```

Reply:
Missed a few of those when changing element to value.